benchmarks/report.ml

```diff
@@ -264,6 +264,18 @@ let output_table =
     stats t;
     f t
 
+let rec transpose (l : (string * 'a list) list) : (string * 'a) list list =
+  match l with
+  | (_, _ :: _) :: _ ->
+      List.map ~f:(fun (label, data) -> label, List.hd data) l
+      :: transpose (List.map ~f:(fun (label, data) -> label, List.tl data) l)
+  | (_, []) :: _ | [] -> []
```

Comment:
```suggestion
let rec transpose (l : (string * 'a list) list) : (string * 'a) list list =
  match l with
  | (_, _ :: _) :: _ ->
      List.map ~f:(fun (label, data) -> label, List.hd data) l
      :: (transpose [@tailcall]) (List.map ~f:(fun (label, data) -> label, List.tl data) l)
  | (_, []) :: _ | [] -> []
  [@@tail_mod_cons]
```
This will be more efficient, in case we care.

Reply:
The function is calling itself recursively once, so we don't really care.